src/main/java/mate/academy/Main.java

```diff
@@ -2,16 +2,27 @@
 
 import java.time.LocalDate;
 import java.time.LocalDateTime;
+import java.util.List;
 import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
 import mate.academy.model.MovieSession;
+import mate.academy.model.ShoppingCart;
+import mate.academy.model.Ticket;
+import mate.academy.model.User;
 import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
 import mate.academy.service.MovieSessionService;
+import mate.academy.service.ShoppingCartService;
+import mate.academy.service.UserService;
+import mate.academy.service.impl.CinemaHallServiceImpl;
+import mate.academy.service.impl.MovieServiceImpl;
+import mate.academy.service.impl.MovieSessionServiceImpl;
+import mate.academy.service.impl.ShoppingCartServiceImpl;
+import mate.academy.service.impl.UserServiceImpl;
 
 public class Main {
     public static void main(String[] args) {
-        MovieService movieService = null;
+        MovieService movieService = new MovieServiceImpl();
```

Comment:
```suggestion
        MovieService movieService = (MovieService) INJECTOR.getInstance(MovieService.class);
```

Check other injection please